articles/client-platforms/vuejs/01-login.md

```diff
@@ -11,8 +11,8 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
-  githubUrl: 'https://github.com/auth0-samples/auth0-vue-samples',
   pkgOrg: 'auth0-samples',
+  githubUrl: 'https://github.com/auth0-samples/auth0-vue-samples/00-Starter-Seed',
```

Comment:
@mehreencs87 Gives [404](https://github.com/auth0-samples/auth0-vue-samples/00-Starter-Seed) Please update this to `https://github.com/auth0-samples/auth0-vue-samples/tree/master/00-Starter-Seed`